src/http-util/src/lib.rs

```diff
@@ -128,3 +129,14 @@ pub async fn handle_modify_filter_target(
         Err(e) => (StatusCode::BAD_REQUEST, e.to_string()),
     }
 }
+
+/// Allows dynamic control of the stderr log filter
+#[allow(clippy::unused_async)]
+pub async fn generate_tracing_info() -> impl IntoResponse {
```

Comment:
For consistency with the other handlers:

```suggestion
pub async fn handle_tracing() -> impl IntoResponse {
```